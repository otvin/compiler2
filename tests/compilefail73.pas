program compilefail73(output);
{invalid use of odd}


var r:real;


begin
	r := 4.6;
	writeln(odd(r));
end.
