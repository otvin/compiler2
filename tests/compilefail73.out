[1mtests/compilefail73.pas:10:13: [0m[1m[91merror:[0m Function odd() requires parameter of integer type
    10 | 	writeln(odd(r));